### DISCUSSION TITLE
MileStones for Microsoft Teams

### DISCUSSION THREAD

**Discussion by jasonbvh** [community/community#139925] [New to GitHub]
Hi I am new to GitHub,
I downloaded Milestones for Microsoft Teams sometime last year, the Teams app installed without issue using Teams Admin center
and this was working up until now.
I am not seeing any tabs inside Teams client when I launch Milestones and one user mentioned it said it needs an update.
My question is , how do I update Milestones for Microsoft Teams from GitHub,?
I don't see any new updates in GitHub unless I am looking in the wrong place and the one I did find says its been archived?
If there is update, where can I find it and how does one apply the updates? Is it new files that updates the existing app in Microsoft Teams admin center or do I need to remove and re-add the App in Teams admin center?

Any help or guidance is appreciated in advanced :)

Thanks 
Jason

**Comment by BDadmehr0** [comment]
Hi Jason!

Welcome to GitHub and to working with Microsoft Teams apps! It sounds like you’re dealing with an issue related to the Milestones app, so let’s go through how to handle updates.

1. **Check the Repository**: If the Milestones app has been archived on GitHub, it likely means that the original developers are no longer maintaining it. This could explain why you’re not seeing updates or new releases. You can still check the repository for any recent commits or discussions that might indicate whether the app is still being developed or if any forks are available.

2. **Look for Forks**: Sometimes, if a repository is archived, other developers might fork it and continue development. Search GitHub for forks of the Milestones repository to see if anyone has continued the project.

3. **Download the Latest Version**: If you do find a more recent version (either from the original repo or a fork), you’ll need to download it. This usually involves downloading the source code as a ZIP file or cloning the repository.

4. **Update Process**:
   - If you have a new version of the app, you will need to upload it to the Teams Admin Center.
   - **Remove the Old Version**: Before uploading the new version, it’s generally a good idea to remove the existing version from the Teams Admin Center.
   - **Upload the New Version**: Once the old version is removed, upload the new app package via the Teams Admin Center.

5. **Testing**: After updating, it’s advisable to test the app with a couple of users to ensure everything works as expected.

6. **Community Support**: If you're stuck, consider reaching out in community forums related to Microsoft Teams development. There might be other users experiencing similar issues or who have insights into using Milestones.

If the app is crucial for your team and there are no updates available, you might also want to consider alternatives or reach out to the developers for support.

I hope this helps, and let me know if you have more questions!

**Comment by jasonbvh** [comment]
Thanks for the update, it is archived so I am guessing that answer my question.

**Reply by BDadmehr0** [reply]
I hope my answer helps you – and if you liked it, please vote for it.
